pyproject.toml

```diff
@@ -10,7 +10,7 @@ authors = []
 
 [tool.poetry.dependencies]
 python = "^3.10"
-ops = "^2.11.0"
+ops = "^2.17.0"
```

Comment:
reason: `ops` did not previously have support for `type: secret` in their testing framework.